src/internal-components/ColumnHeaderCell.tsx

```diff
@@ -63,10 +64,10 @@ export const ColumnHeaderCell: React.FunctionComponent<ColumnHeaderCellProps> =
     } : null;
 
     return (
-        <th {...(props.columnDefinition.tdProps != null ? props.columnDefinition.tdProps : {})}>
+        <th {...(props.columnDefinition.tdProps != null ? props.columnDefinition.tdProps : {})} style={{width: props.columnDefinition.columnWidth ? props.columnDefinition.columnWidth : ''}}>
```

Comment:
It might be nice to, instead of providing the `columnWidth` prop, provide a `className` prop, and forward that to here. Same thing on the `td`s. That way if you want to provide some other value (maybe some padding or something) we don't have to add those on